src/Storefront/Theme/Message/DeleteThemeFilesHandler.php

```diff
@@ -4,12 +4,17 @@
 
 use League\Flysystem\FilesystemOperator;
 use Shopware\Core\Framework\Adapter\Cache\CacheInvalidator;
+use Shopware\Core\Framework\Feature;
 use Shopware\Core\Framework\Log\Package;
 use Shopware\Storefront\Theme\AbstractThemePathBuilder;
 use Symfony\Component\Messenger\Attribute\AsMessageHandler;
 
 /**
  * @internal
+ *
+ * @deprecated tag:v6.8.0 - Will be removed. Unused theme files are now deleted with a scheduled task.
```

Comment:
also this is internal, I decided to not delete it directly as the handled message is not internal :see_no_evil: 

So one could still dispatch the message and not handling it anymore is a break, imo